android/orblibrary/src/main/java/org/orbtv/orblibrary/IOrbSessionCallback.java

```diff
@@ -901,51 +838,12 @@ default void onReceiveError(int code, String message,
     /**
      * @since 204
      *
-     * Request for the description of the current media on applications
+     * Prints the log message using Logcat with a default log level of log.d.
+     * Overrides if a window console exists.
      *
-     * @return true if this event has been handled, and false if not
+     * @param log The log message to be printed
      */
-    default boolean onRequestMediaDescription() {
-        throw new UnsupportedOperationException("Unsupported 204 API.");
-    }
-
-    /**
-     * @since 204
-     *
-     * Request to deliver a text input, from voice command, to applications
-     *
-     * @param input The content of the text
-     * @return true if this event has been handled, and false if not
-     */
-    default boolean onRequestTextInput(String input) {
-        throw new UnsupportedOperationException("Unsupported 204 API.");
-    }
-
-    /**
-     * Called to send an intent, from a voice command, to applications
-     *
-     * @param action The index number of the intent, from intent.media.pause to intent.playback
-     * @param info   The value uniquely identifying a piece of content:
-     *               - INTENT_MEDIA_SEEK_WALLCLOCK: a wall clock time
-     *               - INTENT_DISPLAY: a URI
-     *               - INTENT_SEARCH: a search term specified by the user.
-     *               - INTENT_PLAYBACK: a URI
-     * @param anchor The value indicates an anchor point of the content, which is either "start" or "end"
-     * @param offset The number value for the time position, a number of seconds
-     * @return true if this event has been handled, and false if not
-     */
-    default boolean onSendIntentByVoiceCommand(Integer action, String info, String anchor, int offset) {
-        throw new UnsupportedOperationException("Unsupported 204 API.");
-    }
-
-    /**
-     * @since 204
-     *
-     * Called to send a send a key press event, from a voice command, to the application 
-     *
-     * @param action The index number of the intent, either pressing a button or showing a log
-     */
-    default boolean onSendKeyPressAction(Integer action) {
-        throw new UnsupportedOperationException("Unsupported 204 API.");
+    default void consoleLog(String log) {
```

Comment:
If this is only used for 204, can we throw an exception if it's not implemented, and implement it in the mock204app instead, to be consistent with other methods.